courses/guidance/04.md

```diff
@@ -0,0 +1,29 @@
+---
+index: 4
+title: 4. Starting Your First Vulcan Project
+author: Sumaa Sayed
+headerDepth: 3
+---
+
+## 4. Starting The Project
+
+### 4.1
+- Now you can start your project, after outlining the needs of your application and choosing an SRG.
+
+- At the top of the page, click the "Start New Project" button. 
+
+![Alt text](../../assets/img/Vulcan_Main_Page.png)
+
+- Name your project, give it a description, and choose its visibility.
```

Comment:
might be worth to know that we can add a slack channel at the project level on project creation (optional). Also, admin can configure/change the visibility of a project after creation.
Not sure if necessary, but maybe touch up on private vs discoverable project and highlight the feature that users not part of a project can request to join a discoverable project.

Reply:
Good point.